homeassistant/components/letpot/quality_scale.yaml

```diff
@@ -29,7 +29,7 @@ rules:
   unique-config-entry: done
 
   # Silver
-  action-exceptions: todo
+  action-exceptions: done
```

Comment:
From quick looking at where the integration raises exceptions I think all are now translated, if this is the case you can also set:
https://github.com/home-assistant/core/blob/d1e2a894fa1402482be1d18bf65c882272045263/homeassistant/components/letpot/quality_scale.yaml#L66

to `done` so you don't have to review it again later.

Reply:
You're right, I believe that I had some untranslated ones during development but they were scrapped so this has probably been done since the integration was merged.